installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3277,4 +3277,5 @@ UPDATE cf_Target SET displayName='SQL Server' WHERE targetName in ('cloudsql_sql
 UPDATE cf_Target SET displayName='IAM User' WHERE targetName in ('iamuser','iamusers');
 UPDATE cf_Target SET displayName='KMS Key' WHERE targetName in ('kms','kmskey');
 UPDATE cf_Target SET displayName='Subnet' WHERE targetName in ('subnet','subnets');
-UPDATE cf_Target SET displayName='VM' WHERE targetName in ('virtualmachine','vminstance');
\ No newline at end of file
+UPDATE cf_Target SET displayName='VM' WHERE targetName in ('virtualmachine','vminstance');
+UPDATE cf_Target SET displayName = 'Environment', endpoint = concat(@eshost,':',@esport,'/checkmarx_environment') WHERE dataSourceName in ('checkmarx');
```

Comment:
lets use targetName in where clause, because a single datasource can have multiple targetNames.

Reply:
I'd suggest updating by the primary key so that it behaves consistently at all times, even if another asset type is added in future

Reply:
@ershad-paladin done the changes Please check now